ProcessOptimizer/model_systems/poly2.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+from .model_system import ModelSystem
+
+
+def poly2_score(x):
+    """
+    A simple 2D polynomial with one minimum.
+
+    Intended parameter Space during benchmark use:
+        [(-1.0, 1.0), (-1.0, 1.0)]
+        If x contains more than two dimensions the excess are ignored.
+
+    Global minimum value, f(x*): -2.0512
+    Global mimimum location, x*: (0.6667, -0.4833)
+    Global maximum value, f(x+): -1.270
+    Global maximum location, x+: (-1.0, -1.0)
+
+    Parameters
+    ----------
+    * 'x' [array of floats of length >=2]:
+        The point to evaluate the function at.
+
+    Returns
+    -------
+    * 'score' [float]:
+        The score of the system at x.
+    """
+    return -(
+        2
+        - 0.2 * ((x[0] - 0.3) ** 2 + (x[1] - 0.1) ** 2)
+        + 0.05 * x[0]
+        - 0.1 * x[1]
+        - 0.2 * x[0] * x[1]
+    )
+
+
+poly2 = ModelSystem(
+    poly2_score,
+    [(-1.0, 1.0), (-1.0, 1.0)],
+    noise_model="constant",
```

Comment:
```suggestion
    noise_model={"model_type": "constant", "noise_size": 0.007812},
```
Assigns a default noise of 1 % of the system scale.

Reply:
1% has been set as default noise:
```
model_systems.poly2.noise_size
```
returns 0.007812000000000001